src/components/style/zeroState.css

```diff
@@ -0,0 +1,3 @@
+.pointer-events-none {
+  pointer-events: none;
+}
```

Comment:
this class was originally `.zeroState`. Given that this looks like it could easily be a utility class, what I like to do is just make it a utility class and then it can just be moved into a base class folder easily later (it sort of greases the wheel for adding utility classes which is why I do this, but don't move it immediately to blow out the scope of the PR)

Reply:
decided in meeting: aligned, decide which category

Reply:
we can replace this custom style with the utility class as a separate PR if you want - just file a follow up ticket before merging